styles.css

```diff
@@ -382,8 +435,372 @@ textarea {
+
+  .icon-mobile.active .bar:nth-of-type(3) {
+    transform: translateY(-6.3px) rotate(-45deg);
+  }
+}
+
+/* Start styling desktop version */
+@media (min-width: 768px) {
+  .mobile-home {
+    background-image: none;
+    display: flex;
+    width: 75.8%;
+    font-size: 100%;
+    margin-right: 14%;
+    margin-left: 12%;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 5%;
+    letter-spacing: 10%;
+  }
+
+  body {
+    background-image: url(assets/images/Header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    margin: 0;
+    padding: 0;
+  }
+
+  .header {
+    display: flex;
+    flex-direction: row;
+    gap: 30px;
+    font-size: 100%;
+    justify-content: center;
+    align-items: baseline;
+    margin: 0% auto;
+  }
+
+  .header li a {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-size: 85%;
+    font-weight: 700;
+    cursor: pointer;
+    color: #172b4d;
+  }
+
+  .logo {
+    font-family: "Inter", Helvetica, Arial, sans-serif;
+    font-size: 1.125rem;
+    transform: rotate(360deg);
+    transition: 1s;
+    margin-right: 26%;
+  }
+
+  .nav-menu {
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+    font-weight: 600;
+    font-size: 1rem;
+    line-height: 1.25rem;
+    margin-right: 22rem;
+  }
+
+  .nav-item {
+    padding: 0.625rem 0.75rem;
+  }
+
+  .icon-mobile {
+    display: none;
+  }
+
+  .main-container {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin: 1%;
+    width: 98%;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .primary {
+    margin: -4% 1% 1% 0%;
+    font-weight: 650;
+  }
+
+  .half-primary {
+    font-weight: 650;
+  }
+
+  .supporting-text {
+    margin: 1% 19% 2% 20%;
+    font-weight: 450;
+    line-height: 120%;
+    text-align: center;
+  }
+
+  .social-media-icons li {
+    margin: 2% 1%;
+  }
+
+  .action {
+    color: #505f79;
+  }
+
+  .icon-chevron-down-larger {
+    margin-top: 5%;
+  }
+
+  .primary-text {
+    font-weight: 650;
+    font-size: 340%;
+    text-align: center;
+    width: 180%;
+    margin: 0%;
+    text-decoration: none;
+  }
+
+  /* My Recent Works */
+
+  .card-work-row1 {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    grid-template-rows: auto auto;
+    justify-content: space-evenly;
+    justify-items: stretch;
+  }
+
+  .card-text {
+    flex: 1.5;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    border: 5px solid transparent;
+    border-radius: 4px;
+    align-items: center;
+    width: 100%;
+    background: #fff;
+    color: #fff;
+  }
+
+  button:hover {
+    background: #36b37f;
+    box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+  }
+
+  button:active {
+    transition: all 0.15s;
+    background: #008552;
+    box-shadow: none;
+  }
+
+  /* About me Part stasrts */
+  .about-me {
```

Comment:
- [ ] For a better user experience, Kindly make sure your languages, frameworks, and skills cards are well-centered as is the case in the original design.
- [ ] And even the contact me section is not well-centered, Kinldy consider correcting that.

### Your design :point_down: 
![Screenshot from 2023-06-09 15-16-47](https://github.com/yesuf023/Portfolio/assets/97100091/394c28e1-e011-4385-bc21-e95e2108393a)

### Orginal template :point_down: 
![image](https://github.com/yesuf023/Portfolio/assets/97100091/3de56087-a840-49f8-878b-b44e94a0ee8f)